### DISCUSSION TITLE
Example for sending document

### DISCUSSION THREAD

**Discussion by seamuswc** [atipugin/telegram-bot-ruby#267] [Q&A]
In the docs there is an example for sending a photo:

    bot.api.send_photo(chat_id: message.chat.id, photo: Faraday::UploadIO.new('~/Desktop/jennifer.jpg', 'image/jpeg'))

Could someone provide an example for a document file, or point me to the method that handles it in source code.

Thank you.

**Comment by atipugin** [comment]
Hi @seamuswc,

use [sendDocument](https://core.telegram.org/bots/api#senddocument) method:

```ruby
bot.api.send_document(chat_id: message.chat.id, document: Faraday::UploadIO.new('/Users/atipugin/Desktop/1.pdf', 'application/pdf'))
```
